plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -131,13 +133,11 @@
       <project-facet-version facet="com.google.cloud.tools.eclipse.facet" version="1">
           <constraint>
             <and>
-              <or>
-                  <requires facet="jst.web" version="2.2"/>
-                  <requires facet="jst.web" version="2.3"/>
-                  <requires facet="jst.web" version="2.4"/>
-                  <requires facet="jst.web" version="2.5"/>
-              </or>
-              <requires facet="java" version="1.7]"/>
+               <requires
+                     facet="jst.web"
+                     version="[2.2-2.5]">
+               </requires>
+              <requires facet="java" version="[1.4-1.7]"/>
```

Comment:
This seems exactly the opposite of the PR description. Maybe [1.7]? I guess the bug was that the opening square bracket was missing?